components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint/src/main/webapp/username-recovery.jsp

```diff
@@ -208,8 +208,7 @@
                             <%}%>
 
                             <%
-                                String callback = Encode.forHtmlAttribute
-                                        (request.getParameter("callback"));
+                                String callback = request.getParameter("callback");
```

Comment:
Why was  Encode.forHtmlAttribute removed?

Reply:
when callback is null, Encode.forHtmlAttribute will return the value as "null", which was affecting the following line where user portal URL will be assigned to the call back when it is null.